server/src/repositories/asset.repository.ts

```diff
@@ -572,14 +572,12 @@ export class AssetRepository implements IAssetRepository {
   }
 
   getRandom(ownerId: string, count: number): Promise<AssetEntity[]> {
```

Comment:
Could you add a `@GenerateSql` and generate the sql files while at it? :)

Reply:
Will do.